src/components/FieldHelp/index.jsx

```diff
@@ -48,12 +49,12 @@ class FieldHelp extends React.Component {
       <span id={this.props.id} className={this.props.className}>
         <button
           type="button"
-          className="btn btn-link py-0 px-1 align-baseline"
+          className={`btn btn-link py-0 px-1 align-bottom ${this.props.optional ? 'text-gray-700' : ''}`}
           onClick={this.toggleToolTip}
           onBlur={this.closeToolTip}
           aria-describedby={`${this.props.id}-tooltip`}
         >
-          <span className="fa fa-info-circle" />
+          <InfoOutline />
```

Comment:
This adds a weird margin space between text and the info icon where the mouse treats the space as info icon. I noticed this while testing on the local.

Reply:
fixed